src/sentry/tasks/relocation.py

```diff
@@ -302,14 +302,18 @@ def uploading_start(uuid: UUID, replying_region_name: str | None, org_slug: str
     name="sentry.relocation.fulfill_cross_region_export_request",
     queue="relocation",
     autoretry_for=(Exception,),
-    max_retries=MAX_FAST_TASK_RETRIES,
-    retry_backoff=RETRY_BACKOFF,
+    # So the 1st retry is after ~0.5 min, 2nd after ~1 min, 3rd after ~2 min, 4th after ~4 min.
+    max_retries=4,
```

Comment:
Should we make these constants instead of magic numbers?

Reply:
This is the only task that uses these specific numbers, so we'll just have `max_retries=MAX_RETRIES_FOR_FULFILL..`, which seems a bit redundant. I agree that if we used this constant in multiple places that would be a good way to go.